src/libixy-vfio.c

```diff
@@ -301,7 +299,7 @@ uint64_t vfio_map_dma(void* vaddr, uint32_t size) {
 	struct vfio_iommu_type1_dma_map dma_map = {
 		.vaddr = (uint64_t) vaddr,
 		.iova = iova,
-		.size = size < MIN_DMA_MEMORY ? MIN_DMA_MEMORY : size,
+		.size = size,
```

Comment:
don't we still need this anyways? the function is part of the public interface of libixy-vfio, someone else might try to use it to allocate less than 4kb of DMA memory (which is not completely unrealistic; think of memory needed for the queues)

Reply:
Fair enough, fixed.